apps/screenshot-diff/src/computeScreenshotDiff.ts

```diff
@@ -229,14 +260,28 @@ export const computeScreenshotDiff = async (
     .findById(screenshotDiff.id)
     .patch({ jobStatus: "complete" });
 
+  // Patch group on screenshot diffs
+  const diffsGroups = await ScreenshotDiff.query()
+    .distinctOn("group")
+    .where({ buildId })
+    .whereNotNull("group");
+
+  await Promise.all(
+    diffsGroups.map(async (diffsGroup) =>
+      ScreenshotDiff.query()
+        .where({ buildId, s3Id: diffsGroup.s3Id, group: null })
+        .patch({ group: diffsGroup.group }),
+    ),
+  );
```

Comment:
We don't want to patch all groups, just the group concerning our screenshot diff.

```ts
const similarDiffCount = await ScreenshotDiff.query().where({ buildId, s3Id: screenshotDiff.s3Id }).resultSize()
if (similarDiffCount > 1) {
  await ScreenshotDiff.query().where({ buildId, s3Id: screenshotDiff.s3Id, group: null }).patch({ group: screenshotDiff.s3Id });
}
```